sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/RowToPubsubMessage.java

```diff
@@ -38,29 +37,28 @@
  * <p>Currently only supports writing a flat schema into a JSON payload. This means that all Row
  * field values are written to the {@link PubsubMessage} JSON payload, except for {@code
  * event_timestamp}, which is either ignored or written to the message attributes, depending on
- * whether {@link PubsubJsonTableProvider.PubsubIOTableConfiguration#getTimestampAttribute()} is
- * set.
+ * whether config.getValue("timestampAttributeKey") is set.
  */
 @Experimental
 public class RowToPubsubMessage extends PTransform<PCollection<Row>, PCollection<PubsubMessage>> {
```

Comment:
Hm I feel like this also should not be public... but I see that's actually my fault since I did it originally. Would you mind fixing my mistake and making this a `private static` inner class?